src/@next/Modal/ModalStyle.ts

```diff
@@ -12,7 +12,7 @@ export const StyledModalWrapper = styled.div<
   inset: 0;
   background-color: rgba(45, 45, 45, 0.5);
   display: flex;
-  flex-direction: column;
```

Comment:
is this expected? i think by default it should be column instead of a row, if we have the left and right icons then this should be row?